src/app/common/schemas.py

```diff
@@ -42,7 +42,7 @@ class Config:
 class PNGResponse(BaseModel):
     """Response schema for the create_pngs endpoint."""
 
-    png_urls: list[str]
+    key: list[str]  # key in the S3 bucket
```

Comment:
Maybe `keys` (plural) would be better than `key` , since we have a list of values, not just a single value?

Reply:
I think this is a good idea!